experimental/yarpl/include/yarpl/Flowables.h

```diff
@@ -44,23 +46,28 @@ class Flowables {
   }
 
   template <typename T>
-  static Reference<Flowable<T>> just(std::initializer_list<T> list) {
-    auto lambda = [ list, it = list.begin() ](
+  static Reference<Flowable<T>> justN(std::initializer_list<T> list) {
```

Comment:
`just(T)` should send a single `T`.  It seemed weird to me to have
  `just(T) (T=initializer_list)` not send a single `initializer_list` (despite that being a bad idea and all).  I dunno, I like that when I read `just()` I can easily tell that it's only one value to send.